django_project/frontend/tests/test_metrics.py

```diff
@@ -54,6 +54,19 @@ def setUp(self):
             5, taxon=self.taxon, user=self.user, property=self.property
         )
 
+        # self.taxon1 = TaxonFactory.create(
```

Comment:
Please remove if it's not needed anymore

Reply:
@dimasciput I have done so